app/gui2/index.html

```diff
@@ -17,6 +17,7 @@
   <body>
     <div id="enso-dashboard" class="enso-dashboard"></div>
     <div id="enso-chat" class="enso-chat"></div>
+    <div id="enso-portal-root" class="enso-portal-root"></div>
```

Comment:
We want to be sure that `.enso-portal-root` goes after the dashboard to stay on top of the whole app, so we put this element explicitly after the `dashboard` and `chat`